package.json

```diff
@@ -38,9 +39,9 @@
     "eslint": "^8.57.0",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.6",
+    "nodemon": "^3.1.3",
```

Comment:
I would use `tsx` which has a watch mode via `--watch`

Reply:
Sure, it would be a dependency less.
I'll leave the decision to @LarsTheGlidingSquirrel

Reply:
I will come back to this later